apps/gallery/style/gallery.css

```diff
@@ -851,6 +851,7 @@ a.filter-faded {
   height: 2.6rem;
   -moz-margin-start: calc(3.9rem - 1.8rem);
   background: url(images/slider-scrubber.png) no-repeat center center;
+  background-size: 100% auto;
```

Comment:
I had to look this up to see why you needed two values.  According to MDN, you can just use the 100% and the second value will be auto by default.  Consider putting the size specification in the background attribute for simplicity.  I think that might be "center/100% center", but I'm not positive. Maybe simpler to keep background-size separate.

Do you understand why the lack of background-size was causing it to come out the wrong size? Is this a bug that needs to be fixed elsewhere in our code?